### PR TITLE
web: Add tabIndex in pagination buttons and enter key event to change…

### LINKED ISSUE #448: Pagination in reactivesearch using ResultList component and 508 compliance
We are using pagination in search result. We use ResultList component. Additional to click on page 2, 3 and to support 508 compliance, we need to use tab in order to go to different pages .
How can we do it?

### PR DESCRIPTION
**Pull request by jyash97**
fixes #448

### CODE REVIEW

File: packages/web/src/components/result/addons/Pagination.js [hunk 1]
```diff
@@ -42,6 +42,8 @@ export default function Pagination(props) {
 					className={className}
 					primary={primary}
 					key={i - 1}
+					tabIndex="0"
+					onKeyPress={event => (event.key === 'Enter' ? props.setPage(i - 1) : null)}
```
Comment (metagrover) [inline packages/web/src/components/result/addons/Pagination.js]: Looks good, but we already have [this helper method](https://github.com/appbaseio/reactivecore/blob/master/src/utils/helper.js#L228-L233) that you can use here.

**Comment by divyanshu013** [comment]
Requires #420 to get correct shade of primary color.

**Comment by divyanshu013** [comment]
Merged [here](https://github.com/appbaseio/reactivesearch/commit/dd4bf9eeb0644ec7d639c2946bd3b55e778f2b98)

File: packages/web/src/components/result/addons/Pagination.js [hunk 2]
```diff
@@ -42,6 +42,8 @@ export default function Pagination(props) {
 					className={className}
 					primary={primary}
 					key={i - 1}
+					tabIndex="0"
+					onKeyPress={event => handleA11yAction(event, props.setPage(i - 1))}
```
Comment (divyanshu013) [inline packages/web/src/components/result/addons/Pagination.js]: When passing a callback, it should be a function so the execution can be delayed. For example here the second parameter should have been a function i.e. `() => props.setPage(...)`

**Pull request closed without merging**